management/attack_protection.go

```diff
@@ -20,10 +20,22 @@ func newAttackProtectionManager(m *Management) *AttackProtectionManager {
 //
 // See: https://auth0.com/docs/secure/attack-protection/breached-password-detection
 type BreachedPasswordDetection struct {
-	Enabled                    *bool     `json:"enabled,omitempty"`
-	Shields                    *[]string `json:"shields,omitempty"`
-	AdminNotificationFrequency *[]string `json:"admin_notification_frequency,omitempty"`
-	Method                     *string   `json:"method,omitempty"`
+	Enabled                    *bool                           `json:"enabled,omitempty"`
+	Shields                    *[]string                       `json:"shields,omitempty"`
+	AdminNotificationFrequency *[]string                       `json:"admin_notification_frequency,omitempty"`
+	Method                     *string                         `json:"method,omitempty"`
+	Stage                      *BreachedPasswordDetectionStage `json:"stage,omitempty"`
+}
+
+// BreachedPasswordDetectionStage is used to specify per-stage configuration options.
+type BreachedPasswordDetectionStage struct {
```

Comment:
👍🏻 

**Note for posterity:** We should rename the current `Stage` to `SuspiciousIpThrottlingStage` in the future.